rascaline/src/calculators/radial_basis/mod.rs

```diff
@@ -28,7 +28,12 @@ pub enum RadialBasis {
     /// The easiest way to create a set of spline points is the
     /// `rascaline.generate_splines` Python function.
     TabulatedRadialIntegral {
+        /// spline points
         points: Vec<SplinePoint>,
+        /// contribution of the central atom. This parameter is only required for LODE
+        /// calculations where the length of `center_contribution` must be the same as
+        /// `max_radial`.
+        center_contribution: Option<Vec<f64>>,
```

Comment:
I made the `center_contribution` an optional argument. Let me know if this is okay of if we should create a new enum entry like `TabulatedRadialIntegralCenter`

Reply:
That's fine for me, but we need to document a bit better what "contribution of the central atom" means (i.e. what's the equation that should be computed)